From 0f88992b2681aed4f31dc7dd3926b357bbc95154 Mon Sep 17 00:00:00 2001
From: Tomi Valkeinen <tomi.valkeinen@nokia.com>
Date: Tue, 21 Apr 2009 10:11:55 +0300
Subject: [PATCH] DSS2: DSI: decrease sync timeout from 60s to 2s

The framedone-problem should be ok now, so we shouldn't get long waits.
---
 drivers/video/omap2/dss/dsi.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/drivers/video/omap2/dss/dsi.c b/drivers/video/omap2/dss/dsi.c
index 50af925..d59ad38 100644
--- a/drivers/video/omap2/dss/dsi.c
+++ b/drivers/video/omap2/dss/dsi.c
@@ -3216,7 +3216,7 @@ static void dsi_push_set_mirror(struct omap_display *display, int mirror)
 
 static int dsi_wait_sync(struct omap_display *display)
 {
-	long wait = msecs_to_jiffies(60000);
+	long wait = msecs_to_jiffies(2000);
 	struct completion compl;
 
 	DSSDBGF("");
-- 
1.5.6.5

